packages/atomic/tsconfig.json

```diff
@@ -6,11 +6,11 @@
     "experimentalDecorators": true,
     "lib": [
       "dom",
-      "es2017"
+      "es2020"
```

Comment:
Bumped in order to get the latest changes for that issue https://github.com/microsoft/TypeScript/issues/38012
PR adds to ES2020 and not ES2017 https://github.com/microsoft/TypeScript/pull/38013

I could also roll it back and use declaration merging for the NumberFormatOptions

Reply:
I think it should be good for all modern browser, so...